Userland/Libraries/LibWeb/Layout/TableFormattingContext.h

```diff
@@ -61,12 +62,22 @@ class TableFormattingContext final : public FormattingContext {
         double percentage_width { 0 };
     };
 
+    enum class RowType {
```

Comment:
since this exactly matches `ColumnType` maybe we can reuse single enum definition for both rows and column?

Reply:
It is tempting, otoh they're still different things and wanted them spelled out explicitly. Maybe I can do some type aliases for that.

Reply:
Actually it's a size in both cases after all, fixed.